### PR TITLE
docs: update quickstart_guide.md

### PR DESCRIPTION
**Pull request by jenworthington**

### CODE REVIEW

File: docs/gettingstarted/quickstart_guide.md [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
+    * `netdlp`
+    * `netdns`
+    * `netfw`
+    * `netids`
+    * `netops`
+    * `netwaf`
+    * `netproxy`
+    * `netipam`
+    * `oswinsec`
+    * 'osnix`
```
Comment (mstopa-splunk) [inline docs/gettingstarted/quickstart_guide.md:17]:
```suggestion
    * `osnix`
```

File: docs/gettingstarted/quickstart_guide.md [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
+    * '_metrics` (Optional opt-in for SC4S operational metrics; ensure this is created as a metrics index)
```
Comment (mstopa-splunk) [inline docs/gettingstarted/quickstart_guide.md:18]:
```suggestion
    * `_metrics` (Optional opt-in for SC4S operational metrics; ensure this is created as a metrics index)
```

File: docs/gettingstarted/quickstart_guide.md [hunk 3]
```diff
@@ -1,50 +1,50 @@
...
+
+ 2. Create a HEC token for SC4S. When filling out the form for the token, leave the “Selected Indexes” pane blank and specify that a
+ `lastChanceIndex` be created so that all data received by SC4S will have a target destination in Splunk.
 
 ### SC4S setup (using RHEL 7.6)
-* Set the host OS kernel to match the default receive buffer of sc4s which is set to 16MB
-    * Add following to /etc/sysctl.conf
+1. Set the host OS kernel to match the default receiver buffer of SC4S, which is set to 16MB.
```
Comment (mstopa-splunk) [inline docs/gettingstarted/quickstart_guide.md:24]: step 1 consists of two sub-steps:

```
1. Set the host OS kernel to match the default receiver buffer of SC4S, which is set to 16MB.
- Add following...
- Apply to the kernel...
2. Ensure the kernel is not dropping packets:
```

File: docs/gettingstarted/quickstart_guide.md [hunk 4]
```diff
@@ -1,50 +1,50 @@
...
+2. Add following to /etc/sysctl.conf:
```
Comment (mstopa-splunk) [inline docs/gettingstarted/quickstart_guide.md:25]: do we put it in the inline code mark?

```suggestion
2. Add following to `/etc/sysctl.conf`:
```

File: docs/gettingstarted/quickstart_guide.md [hunk 5]
```diff
@@ -84,19 +84,19 @@
     ```sudo systemctl start sc4s```
 
   
-* Check podman/docker logs for errors (choose one in command below)
+11. Check podman/docker logs for errors (choose one in command below)
```
Comment (mstopa-splunk) [inline docs/gettingstarted/quickstart_guide.md:87]:
```suggestion
11. Check podman/docker logs for errors (choose one in command below):
```

File: docs/gettingstarted/quickstart_guide.md [hunk 6]
```diff
@@ -84,19 +84,19 @@
...
 
     ```
     sudo podman|docker logs SC4S
     ```
   
-* Search on Splunk for successful installation of SC4S
+12. Search on Splunk for successful installation of SC4S
```
Comment (mstopa-splunk) [inline docs/gettingstarted/quickstart_guide.md:93]: colon too?
```suggestion
12. Search on Splunk for successful installation of SC4S:
```

**Comment by mstopa-splunk** [comment]
@jenworthington ready for the final pass

File: docs/gettingstarted/quickstart_guide.md (hunk 3 above)
Comment (jenworthington) [reply docs/gettingstarted/quickstart_guide.md:24]: updated!

File: docs/gettingstarted/quickstart_guide.md [hunk 7]
```diff
@@ -1,50 +1,50 @@
...
+4. Ensure the kernel is not dropping packets:
 
     ```
     netstat -su | grep "receive errors"
     ```
 
-* Create the systemd unit file `/lib/systemd/system/sc4s.service`. Copy and paste from the
-[SC4S sample unit file (Docker)](docker-systemd-general.md#unit-file) or [SC4S sample unit file (Podman)](podman-systemd-general.md#unit-file) .
+5. Create the systemd unit file `/lib/systemd/system/sc4s.service`. Copy and paste from the
+[SC4S sample unit file (Docker)](docker-systemd-general.md#unit-file) or [SC4S sample unit file (Podman)](podman-systemd-general.md#unit-file) :
```
[the remark this answers is not in the document]
Comment (jenworthington) [reply docs/gettingstarted/quickstart_guide.md]: updated, but not 100% sure i got this right...

**Pull request merged**